.travis.yml

```diff
@@ -20,7 +20,7 @@ before_cache:
   - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
   - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 
-script: ./gradlew check
```

Comment:
AFAIK, This will remove running the tests which is probably not intentional.

Reply:
I think the tests still run (can look at the build for this PR). I left the `check` task here, and added `assemble`.

Reply:
Oh sorry, I had missed that since Github highlighted the `assemble` part.